### PR TITLE
Added some Papers

### PR DESCRIPTION
**Pull request by R-Mahmoudi**
Added some Papers

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -602,6 +602,11 @@ P.P.S, Looking for guides and interviews on applying ML? 👉[`applyingML`](http
 25. [Didact AI: The anatomy of an ML-powered stock picking engine](https://principiamundi.com/posts/didact-anatomy/?utm_campaign=Data_Elixir&utm_source=Data_Elixir_407/) `Didact AI` `2022`
 26. [Deployment for Free - A Machine Learning Platform for Stitch Fix's Data Scientists](https://multithreaded.stitchfix.com/blog/2022/07/14/deployment-for-free/) `Stitch Fix` `2022`
 27. [Machine Learning Operations (MLOps): Overview, Definition, and Architecture](https://arxiv.org/abs/2205.02302) ([Paper](https://arxiv.org/ftp/arxiv/papers/2205/2205.02302.pdf)) `IBM` `2022`
+28. [The ML Test Score: A Rubric for ML Production Readiness and Technical Debt Reduction](https://research.google/pubs/pub46555/) ([Paper](https://static.googleusercontent.com/media/research.google.com/cs//pubs/archive/aad9f93b86b7addfea4c419b9100c6cdd26cacea.pdf)) `Google` `2017`
```
Comment (eugeneyan) [inline README.md:605]: Could you order this paper chronologically please?

File: README.md [hunk 2]
```diff
@@ -602,6 +602,11 @@ P.P.S, Looking for guides and interviews on applying ML? 👉[`applyingML`](http
...
+29. [hidden technical debt in machine learning systems](https://arxiv.org/abs/2103.10510) ([Paper](https://proceedings.neurips.cc/paper/2015/file/86df7dcfd896fcaf2674f757a2463eba-Paper.pdf)) `Google` `2015`
```
Comment (eugeneyan) [inline README.md:606]: Ditto.

File: README.md [hunk 3]
```diff
@@ -602,6 +602,11 @@ P.P.S, Looking for guides and interviews on applying ML? 👉[`applyingML`](http
...
+30. [MLOps: Continuous delivery and automation pipelines in machine learning](https://cloud.google.com/architecture/mlops-continuous-delivery-and-automation-pipelines-in-machine-learning) ([Blog](https://cloud.google.com/architecture/mlops-continuous-delivery-and-automation-pipelines-in-machine-learning)) `Google` `2021`
+31. [Design Patterns in Machine Learning for MLOps](https://towardsdatascience.com/design-patterns-in-machine-learning-for-mlops-a3f63f745ce4) ([Blog](https://towardsdatascience.com/design-patterns-in-machine-learning-for-mlops-a3f63f745ce4)) `Meduim` `2022`
```
Comment (eugeneyan) [inline README.md:608]: We don't need separate links if they're they same link. Also, could you fix the spelling error please?

File: README.md [hunk 4]
```diff
@@ -625,7 +630,7 @@ P.P.S, Looking for guides and interviews on applying ML? 👉[`applyingML`](http
 19. [Automatic Retraining for Machine Learning Models: Tips and Lessons Learned](https://building.nubank.com.br/automatic-retraining-for-machine-learning-models/) `Nubank` `2022`
 20. [RecSysOps: Best Practices for Operating a Large-Scale Recommender System](https://netflixtechblog.medium.com/recsysops-best-practices-for-operating-a-large-scale-recommender-system-95bbe195a841) `Netflix` `2022`
 21. [ML Education at Uber: Frameworks Inspired by Engineering Principles](https://www.uber.com/en-PL/blog/ml-education-at-uber/) `Uber` `2022`
-
+22. [MLOps Best Practices for Data Scientists](https://towardsdatascience.com/mlops-practices-for-data-scientists-dbb01be45dd8)`Meduim` `2021`
```
Comment (eugeneyan) [inline README.md:633]: Ditto.

File: README.md (hunk 1 above)
Comment (R-Mahmoudi) [reply README.md:605]: @eugeneyan  Yes, Sure. Thank you, I will do it.

**Comment by eugeneyan** [comment]
Closing due to lack of activity.

**Pull request closed without merging**